nbodykit/source/catalog/tests/test_file.py

```diff
@@ -148,3 +148,35 @@ def test_stack_list(comm):
 
     os.unlink(tmpfile1)
     os.unlink(tmpfile2)
+
+
+# MyFileType is Used by test_file_type
+from nbodykit.io.base import FileType
+
+class MyFileType(FileType):
```

Comment:
@xiaohanzai

You can also try to follow the example here for adapting the different number of files and mpi ranks. #627 (after installing the nbodykit from this PR).